src/FSharp.Literate/Transformations.fs

```diff
@@ -38,8 +38,12 @@ module internal Transformations =
   /// Replace CodeBlock elements with formatted HTML that was processed by the F# snippets tool
   /// (The dictionary argument is a map from original code snippets to formatted HTML snippets.)
   let rec replaceCodeSnippets path (codeLookup:IDictionary<_, _>) = function
-    | CodeBlock(String.StartsWithWrapped ("[", "]") (ParseCommands cmds, String.TrimStart code)) 
+    | CodeBlock(String.StartsWithWrapped ("[", "]") (ParseCommands cmds, code)) 
     | CodeBlock(Let (dict []) (cmds, code)) ->
+        let code = 
+            if code.StartsWith("\r\n") then code.Substring(2)
+            elif code.StartsWith("\n") then code.Substring(1)
```

Comment:
Do we ever expect to get `\r\n` or just `\r` here? I never quite understood newlines in Markdown :-/. Aside from that, it's certainly better to just delete one new line rather than do an arbitrary trim.